Makefile

```diff
@@ -104,6 +105,15 @@ sqlx-local-clean-sqlite:
 sqlx-prepare:
 	$(foreach crate,$(ALL_DATABASE_CRATES),(cd $(crate) && cargo sqlx prepare );)
 
+###############################################################################
+# Sqlx: add migration
+###############################################################################
+
+.PHONY: sqlx-add-migration
+sqlx-add-migration:
```

Comment:
Why would we need this separately from `sqlx-local-setup` ?

Reply:
It is convenient to have a separate command that will add migrations the moment we need to create them